src/Workspaces/Core/Portable/Shared/Extensions/SemanticModelExtensions.cs

```diff
@@ -295,5 +296,53 @@ public static SemanticModel GetOriginalSemanticModel(this SemanticModel semantic
                 }
             }
         }
+
+        public static ValueUsageInfo GetValueUsageInfo(
+            this SemanticModel semanticModel,
+            SyntaxNode node,
+            ISemanticFactsService semanticFacts,
+            CancellationToken cancellationToken)
```

Comment:
did this move from somewhere?

Reply:
No, it was added here so we don't need to track multiple boolean flags. Additionally, I am hoping that this method will completely go away soon when we merge in [https://github.com/dotnet/roslyn/pull/30123](https://github.com/dotnet/roslyn/pull/30123)

---
In reply to: [220754879](https://github.com/dotnet/roslyn/pull/30155#discussion_r220754879) [](ancestors = 220754879)

Reply:
Added comment [https://github.com/dotnet/roslyn/pull/30123#issuecomment-426397361](https://github.com/dotnet/roslyn/pull/30123#issuecomment-426397361)

---
In reply to: [222061927](https://github.com/dotnet/roslyn/pull/30155#discussion_r222061927) [](ancestors = 222061927)